features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/components/APIDetailsTopMenu.jsx

```diff
@@ -12,8 +12,8 @@ import Chip from '@material-ui/core/Chip';
 import { Link } from 'react-router-dom';
 // import { FormattedMessage } from 'react-intl';
 
-import ImageGenerator from '../../Listing/components/ImageGenerator';
 import DeleteApiButton from './DeleteApiButton';
+import ThumbnailView from '../../Listing/components/ThumbnailView';
```

Comment:
Can you please change the relating import path with aliased path like `AppComponents/Apis/Listing/components/ThumbnailView`